app.py

```diff
@@ -56,30 +56,28 @@ def email_redirect(path):
         account_repo.store_login(data)
         conn.commit()
 
-    # redirect for deprecated endpoint.
-    if data.endpoint == "email_unsubscribe":
-        data.endpoint = "unsubscribe"
-
     return redirect(url_for(data.endpoint, **data.data))
 
 
 @app.route(f"{URL_PREFIX}/enroll", methods=["GET"])
 def pre_enroll_get():
     source = request.args.get("source", DEFAULT_SOURCE)
+    subsource = request.args.get("subsource", DEFAULT_SOURCE)
```

Comment:
do we want a default sub_source seperate from default source?
Or alternatively is it worth plumbing through the code to make this null-safe?

Reply:
I went with the option one.

Reply:
as in make it null safe?